Add: more meme songs

zephiowl opened the pull request:
Thought I'd take the chance to add some of the songs I've converted myself while playing web fishing with this tool!

I'll be happy to make more if anyone wants, just open an [issue](https://github.com/KevAquila/WEBFISHING-Guitar-Player/issues) with it/mention me and I'll add a pr for it <3 

Closes: #13 
Added the song a user was having issues with

zephiowl committed:
Add: more meme songs

Zephira58 commented:
Btw if you want me to redo the pr for the verified signing lmk, i just forgot to update the key i used in my gitconfig lol

zephiowl committed:
Fix: Updated midi encoding

Manually tweaked some of the midi files using FL Studio to play a bit better. And added the correct file extension for all of the songs for compatibility

zephiowl committed:
FIx: Midi encoding

Accidentally pushed the wrong branch last time, this one should have all the updated information correctly <3

zephiowl committed:
Add: I see the light, Broken dreams

Their both pretty popular and i think people would want to play them

zephiowl committed:
Remove: Duplicate Songs

There's some songs which are duplicates of others, I've removed them to save space and make it easier to identify which song should be played.

zephiowl committed:
Add: Scatman

Closes: #13
Adds a song a user was having issues with, which should also close their issue, as the application will ship with the new song

Zephira58 commented:
The latest commit should close #13

zephiowl committed:
Add: Captainsparklez - Fallen Kingdom

ingame user request :3

zephiowl committed:
Add: Bastille - Pompeii

zephiowl committed:
Fix: song name

just made it index the same in the playlist by changing the name to match the existing example

zephiowl commented:
I broke my branches ima redo the PR correctly x.x
new pr here: https://github.com/KevAquila/WEBFISHING-Guitar-Player/pull/32
